### ISSUE TITLE
[BUG] OpenGL Render Error On Load

### ISSUE THREAD

**Issue by LousyBook-01** [CCBlueX/LiquidBounce#4498]
### LiquidBounce Branch

Nextgen

### LiquidBounce Build/Version

since 1.17.1

### Operating System

Windows 11

### Minecraft Version

1.21.1

### Describe the bug

Sometimes i get a opengl render error when launching, when i get the error, i have reinstall liquidbounce, by deleteing the liquidbounce folder in gameDir, when i get the error the screen remains fully black, but some logs do appear after sometime, they are not realted to the black screen i think

### Steps to reproduce

- Open Minecraft 1.21.1 with the Latest Dev Build using Liquid Launcher
- There is a blackscreen after the game loads, instead of the menu

### Client Log

```shell
[13:58:30] [Render thread/ERROR]: ########## GL ERROR ##########
[13:58:30] [Render thread/ERROR]: @ Render
[13:58:30] [Render thread/ERROR]: 65539: Invalid standard cursor 0x00000000
```

### Screenshots

It is a black screen lol

**Comment by MukjepScarlet** [comment]
In my test this log also appeared but it had no effect on the game.

**Comment by 1zun4** [comment]
The error is not relevant and does not cause issues. It's from MCEF and is not related to your actual issue.

**Comment by LousyBook-01** [comment]
ok it was fixed since LiquidBounce 0.18 +
